syntax = "proto3";

package tsumaki;

message ErrorResponse {
    int32 code = 1;
    string msg = 2;
}
